scripts/extension-upload-single.sh

```diff
@@ -65,7 +65,7 @@ if [ -z "$AWS_ACCESS_KEY_ID" ]; then
 fi
 
 # Set dry run unless guard var is set
-DRY_RUN_PARAM="--dryrun"
+DRY_RUN_PARAM="--if-match '1234567890abcdef'"
```

Comment:
Is it intentional that this removes the dryrun param? 

Also a nit: can we comment the intention of this flag in the script? the purpose of the magic value here might not be immediately clear

Reply:
It's intentional, given `--dryrun` is not an allowed parameter for `aws s3api` api.

I will double check this to provide an `if-match` that never matches, and and add a comment for future us.